lerna.json

```diff
@@ -1,8 +1,15 @@
 {
+  "command": {
+    "version": {
+      "allowBranch": [
+        "release/*"
```

Comment:
allow running lerna scripts in release/* branches

Reply:
ah got it. Thanks Ben.